lib/rmagick_internal.rb

```diff
@@ -50,6 +51,11 @@ def trace_proc=(p)
         @trace_proc = p
       end
     end
+
+    def lib_version
```

Comment:
I wonder if this would make more sense as a constant. Something like:

```rb
module Magick
  IMAGEMAGICK_VERSION = Magick::Magick_version.split[1].split('-').first
end
```